### ISSUE TITLE
Current restivus build causes crash on "Restivus.configure"

### ISSUE THREAD

**Issue by Tolmark12** [kahmali/meteor-restivus#1]
Looks like [this line](https://github.com/kahmali/meteor-restivus/blob/master/lib/restivus.coffee#L37) causes meteor to crash when calling `Restivus.configure()` and `config.apiPath` isn't defined.

Can be fixed by adding _existence checks_ :
```coffeescript
  if config.apiPath?[0] is '/'
    config.apiPath = config.apiPath.slice 1
  if config.apiPath?[-1] isnt '/'
    config.apiPath = config.apiPath + '/'
```

**Comment by kahmali** [comment]
Good catch! Sorry about that. Your suggestion would fix the error, but the intended behavior is that the default `apiPath` ('api/') be used in that case. To fix that, I needed to copy all the properties from `config` to `this.config` (for some crazy reason I was doing that immediately _after_ trying to normalize the base api path instead of _before_, where it belonged), and then use `this.config.apiPath` instead of `config.apiPath` during the normalization. Again, sorry if that caused you any headache. I'll have the fix pushed and published very soon. I'll let you know once it's all set.

**Comment by Tolmark12** [comment]
Sweet, thanks! 

btw, Great work on Restivus, just what I needed.

**Comment by kahmali** [comment]
Just published v0.5.4 with the fix. You should be able to grab it with `> meteor update restivus`. Thanks again for reporting it! And thanks for the compliment! It makes me warm and fuzzy inside that Restivus has been helpful to you. This is my first project that anyone besides myself has actually used, so this is all very awesome for me.
